### PR TITLE
BlackoilModelEbos: some janitoring

### PR DESCRIPTION
**Pull request by akva2**
Extracted the hopefully non-controversial parts from https://github.com/OPM/opm-simulators/pull/4509

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/flow/BlackoilModelEbos.hpp [hunk 1]
```diff
@@ -52,24 +51,21 @@
...
 
 #include <algorithm>
 #include <cassert>
 #include <cmath>
 #include <iomanip>
-#include <iostream>
 #include <limits>
 #include <type_traits>
-#include <utility>
+#include <tuple>
```
Comment (bska) [inline opm/simulators/flow/BlackoilModelEbos.hpp:68]: To be honest I think you need `<tuple>` **in addition** to `<utility>` here.  The file uses `std::move()` in a number of places and that function (cast) is defined in `<utility>`.  Moreover, PR #3764 which was merged in the interim has explicit uses of `std::pair<>`, also from `<utility>`.

**Review by bska — COMMENTED** [review]
Looks good to me, except for switching `<utility>` out in favour of `<tuple>` in `BlackoilModelEbos.hpp`.

File: opm/simulators/flow/BlackoilModelEbos.hpp (hunk 1 above)
Comment (akva2) [reply opm/simulators/flow/BlackoilModelEbos.hpp]: yeah, situation has changed. so i simply made it include tuple in addition.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**